README.md

```diff
@@ -1,2 +1,185 @@
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello Microverse <a name="about-project"></a>
+
+**Hello Microverse** is a starting project to test linters and core files like gitignore with GitHub, HTML, CSS, and Javascript
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>VCS</summary>
+  <ul>
+    <li><a href="https://git-scm.com/">Git</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Text Editor</summary>
+  <ul>
+   <li><a href="https://code.visualstudio.com/">Visual Studio Code</a></li>
+  </ul>
+
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Implement Lint configuration files**
+- **Add Readme file with template**
+-**Core files such as HTML, CSS, and Javascript**
+- **Pull request**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- Coming soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+1. Login to GitHub
+2. Copy the repository's [URL](https://github.com/Estete9/hello-microverse/tree/add-core-files)
+3. Navigate to your preferred directory to clone the project
+4. Use the command ```git clone [URL]``` to clone
+5. Open project in a text editor
+
+### Prerequisites
+
+In order to run this project you need:
+- Git
+- GitHub
+- A text editor
+
+### Setup
+
+Clone this repository to your desired folder:
+
+1. Login to GitHub
+2. Copy the [repository's](https://github.com/Estete9/hello-microverse) URL
+3. Navigate to your preferred directory to clone the project
+4. Use the command ```git clone [URL]``` to clone
+5. Open project in a text editor
+
+### Install
+
+Disclaimer: This project does not require installation. Just clone the repository open the HTML file in your favorite browser.
+For general purposes, Install this project with:
+```sh
+  cd my-folder
+  npm install
+```
+
+
+### Usage
+
+To run the project, do the following:
+- Clone the repository
+- open the folder in a code editor.
+- Find the path of the index.html
+- open the index.html file in a Web Browser.
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Esteban Palacios**
+
+- GitHub: [@Estete9](https://github.com/Estete9)
+- Twitter: [@NaughTban](https://twitter.com/NaughTban)
+- LinkedIn: [Esteban Palacios](https://www.linkedin.com/in/esteban-palacios-5030a772/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Coming soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](#) (coming soon).
+
```

Comment:
- [ ] This link redirected me to a wrong page but I guessed you intended to add `https://github.com/Estete9/hello-microverse/issues`,kindly make necessary adjustment. 👍